core/templates/pages/story-editor-page/modal-templates/story-editor-unpublish-modal.component.html

```diff
@@ -65,7 +65,7 @@
                   (keydown.enter)="cancel()">
                   Cancel
           </button>
-          <button class="btn oppia-confirm-reason-button"
+          <button class="btn oppia-confirm-reason-button e2e-test-close-save-modal-button"
```

Comment:
should we call it `e2e-test-confirm-unpublish-modal-button`?

Reply:
Done